ci/tests/general/general_vdisk.py

```diff
@@ -222,3 +229,27 @@ def get_filesystem_location(vpool, vdisk_name):
         else:
             raise RuntimeError('Invalid hypervisor type specified: {0}'.format(hv_type))
         return location
+
+    @staticmethod
+    def get_config_params(vdisk_guid):
```

Comment:
My attempt when updating the tests, was to have as much functions as possible passing objects instead of guids or specific attributes of an object
In the method where you use this functionality you have the vdisk object at your disposal anyway

Same for set_config_params